lightning/src/ln/channelmanager.rs

```diff
@@ -2864,6 +2865,23 @@ macro_rules! send_channel_ready {
 		}
 	}}
 }
+macro_rules! emit_funding_signed_event {
+	($locked_events: expr, $channel: expr) => {
+		let funding_tx = $channel.context.get_funding_transaction(); // hmm.. this is always none?
```

Comment:
the funding_transaction is `None` while it should be defined at this stage as we are after `FundingCreated`.. what could be the reason for this?